operator/pkg/apis/cassandra/v1beta1/cassandradatacenter_types.go

```diff
@@ -46,25 +45,71 @@ const (
 // This type exists so there's no chance of pushing random strings to our progress status
 type ProgressState string
 
+const (
+	defaultConfigBuilderImage     = "datastax/cass-config-builder:1.0.0"
+	cassandra_3_11_6              = "datastax/cassandra-mgmtapi-3_11_6:v0.1.2"
+	cassandra_4_0_0               = "datastax/cassandra-mgmtapi-4_0_0:v0.1.2"
+	dse_6_8_0                     = "datastax/dse-server:6.8.0"
+	ubi_cassandra_3_11_6          = "registry.connect.redhat.com/datastax/cassandra:3.11.6"
+	ubi_cassandra_4_0_0           = "TODO"
+	ubi_dse_6_8_0                 = "registry.connect.redhat.com/datastax/dse-server:6.8.0"
```

Comment:
6.8.1 has been released, should this be updated or a separate PR?

Reply:
This approach doesn't seem very tenable as we move forward with more versions. I'm still of the opinion we should simply override the namespace or registry portion of the coordinates.

**Namespace:**
_datastax_/repo\_name:tag

Here _datastax_ is the namespace. It blindly assumes the repo names are the same.

**Repository**
_datastax/repo\_name_:tag

With this approach, the entire repository must be updated. It will require multiple constants, but it feels a little more organic as new versions of components come out.

My preference would be to store the repositories in constants and choose the appropriate value based on the UBI flag passed into the operator.

Reply:
or even just have environment variables available to override the default and ignore UBI entirely in the codebase.

Reply:
I'm gonna put 6.8.1 in another PR - don't want to do that until this one is merged

Reply:
![Creeper gonna creep](https://user-images.githubusercontent.com/135264/85592638-57641780-b614-11ea-8d63-d0af61d37a1b.png)

This was correctly identified as scope creep. Please ignore for now as private registries will go into a separate issue for consideration later.